### ISSUE TITLE
Copies period that follows OTP codes

### ISSUE THREAD

**Issue by jeharrington0** [Yac-Team/ohtipi#13]
Hello!

Having an issue where it will copy the period at the end of texted codes. For example: Receiving "Your verification code is 999999." will copy "999999." to clipboard. Has this issue already been known to occur, and are there any plans to update? 

Otherwise, really liking how this works. Thanks!

**Comment by itsthisjustin** [comment]
Oh strange never seen that one. Will fix.

**Comment by alectrocute** [comment]
Fixed! Will be released in this week, @jeharrington0!

**Comment by jeharrington0** [comment]
Hi again! 

I gave it some time and a couple re-boots in between but it looks like it is still copying the period. I tried to download again off the site, but it was asking for payment again. I feel like such a novice asking this lol, but how do I update to a version with that element fixed?

Thanks!

![Screen Shot 2022-03-03 at 10 22 45 AM](https://user-images.githubusercontent.com/95499679/156628246-55298a45-3778-427b-8fbd-577aa17c9563.png)
